etl/steps/data/meadow/ophi/2022-12-13/multidimensional_poverty_index.meta.yml

```diff
@@ -1,24 +1,104 @@
+    \ – using two indicators: years of schooling, school attendance\n    <li><strong>Living\
+    \ standards</strong> – using five indicators: cooking fuel, sanitation, drinking\
+    \ water, electricity, housing, assets.</li>\n</ul>\nHouseholds are assessed as\
+    \ being deprived in a given indicator if they do not meet a certain threshold\
+    \ for that indicator. For instance, a household is deprived in the <em>Years of\
+    \ schooling</em> indicator if no household member has completed six years of schooling.\
+    \ A person is considered deprived in the <em>Cooking fuel</em> indicator if they\
+    \ cook using solid fuel, such as dung, agricultural crops, wood, charcoal, or\
+    \ coal. The thresholds for each indicator are published by OPHI in their <a href=\"\
+    https://www.ophi.org.uk/wp-content/uploads/OPHI_MPI_MN_54_2022.pdf\">methodological\
+    \ notes</a>.\n\nThe individual indicators are not ‘weighted’ equally: When adding\
+    \ up the number of indicators in which a person is deprived, some count for more\
+    \ than others. Health and education indicators are given a weight of 1/6, while\
+    \ the indicators within the living standards dimension are given a weight of 1/18.\
+    \ This means that the three dimensions – health, education and living standards\
+    \ – have an equal weight in the total of one-third each.\n\nIf the household survey\
+    \ data being used is missing any of the 10 indicators, that indicator is dropped\
+    \ from the calculation. The weights are then adjusted so that each dimension continues\
+    \ to be given a weight of one-third. MPI poverty estimates are only calculated\
+    \ if at least one indicator in each dimension is available.\n\nThe global MPI\
+    \ was developed by OPHI with the UN Development Programme (UNDP) for inclusion\
+    \ in UNDP’s flagship Human Development Report in 2010. It has been published annually\
+    \ by OPHI and in the HDRs ever since.\n"
   sources:
     -
-      name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
-      published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
-      publisher_source: |
-        The Maddison Project Database is based on the work of many researchers that have produced estimates of...
-
+      name: Alkire, Kanagaratnam and Suppa (2022), The global Multidimensional Poverty Index (MPI) 2022
+      published_by: Alkire, S., Kanagaratnam, U., and Suppa, N. (2022). ‘The global Multidimensional Poverty Index (MPI) 2022 country results and methodological note’, OPHI MPI Methodological Note 52, Oxford Poverty and Human Development Initiative, University of Oxford.
+      publication_year: 2022
+      date_accessed: 2022-10-26
+      url: https://ophi.org.uk/multidimensional-poverty-index/
 tables:
   multidimensional_poverty_index:
     variables:
```

Comment:
If you are not adding any variable metadata, I suppose it's better to not add them in the metadata yaml file. Otherwise it feels like those units you gave are the right units.